src/updateFileFromServer.ts

```diff
@@ -96,14 +120,38 @@ async function getServerData(todoistQuery: string, authToken: string): Promise<s
 		new Notice(errorMsg);
 		throw(e)
 	}
-	if (tasks.length === 0){
-		new Notice(`Todoist text: You have no tasks matching filter "${todoistQuery}"`);
+	return tasks;
+}
+
+function getSubTasks(subtasks: Task[], parentId: string, indent: number): string {
+	let returnString = "";
+	let filtered = subtasks.filter(sub => sub.parentId == parentId);
+	if (filtered.length > 0) {
+		filtered.map(st => {
```

Comment:
it's preferable to use `forEach` instead, since the `map` isn't returning anything - https://stackoverflow.com/a/65286561

also, once you switch to `forEach`, you'll be able to remove the `if length > 0` check, b/c the `forEach` would just no-op if empty

Reply:
Ah, got it.  Thanks for the SO link; it helped cement the reason behind the different usages, which I hadn't figured out yet.  I'm coming from C# so it's similar, but I'm a little rusty.  And there's always little details to pick up when learning a new language, whoop!  :)